core/store/src/utils/mod.rs

```diff
@@ -285,6 +299,37 @@ pub fn get_access_key(
     )
 }
 
+pub fn get_gas_key(
+    trie: &dyn TrieAccess,
+    account_id: &AccountId,
+    public_key: &PublicKey,
```

Comment:
If we will clone them why even pass it in as reference? I understand that `get_access_key` follows the same convention, but it might be better to fix it as well in a separate PR instead of adding more cases like this.